packages/network/src/logic/node/Node.ts

```diff
@@ -202,8 +214,8 @@ export class Node extends EventEmitter {
             if (sendStatus) {
                 this.trackerManager.sendStreamStatus(spid)
             }
-        } else if (this.streams.isSetUp(spid) && this.streams.isOneDirectional(spid)) {
-            logger.trace(`Could not join stream ${spid.key} as stream is set as one-directional`)
+        } else if (this.streams.isSetUp(spid) && this.streams.isBehindProxy(spid)) {
```

Comment:
```suggestion
        } else if (this.streams.isBehindProxy(spid)) {
```
The first conditional expression is unnecessary as we are in the `else` clause therefore we know that `this.streams.isSetUp(spid)` must be true.